DataFormats/SiStripCluster/interface/SiStripClusterTools.h

```diff
@@ -29,7 +29,7 @@ namespace siStripClusterTools {
 
   template<typename Clus>
   float chargePerCM(DetId detid, Clus const & cl) {
```

Comment:
it is ill defined...
Clus was supposed to be a Cluster not a ClusterRef
@mtosi  please use 
```
siStripClusterTools::chargePerCM(detid, *cluster, LV);
```
and do not modify this template function...
I may have to add some SFINAE trick